File: Models/Clinic.cs
using System;
using System.Collections.Generic;

namespace Peohe.Models
{
    public class Clinic
    {
        public int ClinicId { get; set; }
        public string Name { get; set; }
        public double Percentage { get; set; }
        public IEnumerable<ClinicDoctor> ClinicDoctors { get; set; }
        public IEnumerable<Attendance> Attendances { get; set; }
        public DateTime? Deleted { get; set; }
    }
}

File: ClientApp/src/app/clinic/clinic.ts
import { ClinicDoctor } from "../models/ClinicDoctor";
import { Attendance } from "../attendance/attendance";

export class Clinic {
  clinicId: string;
  name: string;
  percentage: number;
  clinicDoctors: ClinicDoctor[];
  attendances: Attendance[];
  deleted: string;
  aplicationUserId: string;
}
